pokete_classes/roadmap.py

```diff
@@ -46,12 +48,12 @@ def unchoose(self):
         """Unchooses the station"""
         self.rechar(self.color + self.org_char + Color.reset)
 
-    def next(self, _ev):
+    def next(self, inp):
         """Chooses the next station in a certain direction
         ARGS:
             _ev: Event object"""
```

Comment:
The docstring needs to be updated here as well:
```suggestion
            inp: Event object"""
```